config/findbugs-filter.xml

```diff
@@ -72,6 +72,18 @@
     <Match>
         <Bug pattern="SE_COMPARATOR_SHOULD_BE_SERIALIZABLE" />
     </Match>
+    <Match>
+        <Bug pattern="ES_COMPARING_STRINGS_WITH_EQ" />
```

Comment:
What is adding these `Match` nodes doing here? I'm not very familiar with Spotbugs. Is this adding extra checks?

Reply:
Since the report level of spotbugs is changed to`low`, so new checks are added. Using the above filters I have excluded those newly added checks which haven't been fixed yet